tests/test_materials.py

```diff
@@ -0,0 +1,71 @@
+from mock.mock import MagicMock
```

Comment:
Optional: Add 'get' test for an empty custom materials dict

Reply:
Done